safe/gui/tools/wizard/test/test_keyword_wizard.py

```diff
@@ -1786,6 +1788,149 @@ def test_exposure_multi_fields_existing_keyword(self):
 
         self.assertDictEqual(byteify(real_keywords), byteify(expected_keyword))
 
+    def test_exposure_place_population(self):
+        """Test for place with population exposure."""
+        layer = load_test_vector_layer(
+            'gisv4',
+            'exposure',
+            'places.geojson',
+            clone_to_memory=True)
+        self.assertIsNotNone(layer)
```

Comment:
No worries, load_test_vector_layer will raise an exception if the layer is not valid ;-)